src/Features/Core/Portable/DesignerAttribute/AbstractDesignerAttributeIncrementalAnalyzer.cs

```diff
@@ -108,27 +118,27 @@ public DesignerAttributeDiscoveryService()
             // Now get all the values that actually changed and notify VS about them. We don't need
             // to tell it about the ones that didn't change since that will have no effect on the
             // user experience.
-            var changedData = await ComputeChangedDataAsync(
-                project, specificDocument, projectVersion, designerCategoryType, cancellationToken).ConfigureAwait(false);
 
-            // Only bother reporting non-empty information to save an unnecessary RPC.
```

Comment:
this isn't necessary anymore.  by definition, if we don't yield something, we have nothing to send over the rpc channel.